test/helpers/test-app.js

```diff
@@ -3,7 +3,7 @@
 const path = require('path');
 const rimraf = require('rimraf');
 const execa = require('execa');
-const generateNew = require('../../packages/generators/app/lib/generate-new');
+const generateNew = require('../../packages/generators/app/dist/generate-new').default;
```

Comment:
I haven't changed that and copied it over from https://github.com/strapi/strapi/pull/16392/files#diff-766f34c8554172936b6faf5c8c7f02a5a42f7028938a2117f71d548b7ace1116R5. it doesn't seem like it can be imported from index